pkg/generate/app/sourcelookup.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*\\.git(?:#([a-zA-Z0-9]*))?$")
+	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*(?:#([a-zA-Z0-9]*))?$")
```

Comment:
Hrm - this means we cannot distinguish between URLs and Repos anymore.  Let me think about how I feel about this.